src/main/java/ch/njol/skript/Skript.java

```diff
@@ -518,42 +513,23 @@ public void onEnable() {
 			@Override
 			public void run() {
 				assert Bukkit.getWorlds().get(0).getFullTime() == tick;
```

Comment:
What is this actually for? It's a test-only action but I can't see a comment explaining it.